### ISSUE TITLE
Docker image error: 'No module named material'

### ISSUE THREAD

**Issue by mattbostock** [squidfunk/mkdocs-material#277]
## Description

The official Docker image returns an error:

```
» docker run --rm -it -v `pwd`:/docs squidfunk/mkdocs-material
INFO    -  Building documentation...
Traceback (most recent call last):
  File "/usr/bin/mkdocs", line 11, in <module>
    load_entry_point('mkdocs==0.16.3', 'console_scripts', 'mkdocs')()
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click-6.7-py2.7.egg/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/__main__.py", line 127, in serve_command
    livereload=livereload
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/commands/serve.py", line 110, in serve
    config = builder()
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/commands/serve.py", line 101, in builder
    theme_dir=theme_dir
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/config/base.py", line 181, in load_config
    errors, warnings = cfg.validate()
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/config/base.py", line 105, in validate
    post_failed, post_warnings = self._post_validate()
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/config/base.py", line 85, in _post_validate
    config_option.post_validation(self, key_name=key)
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/config/config_options.py", line 288, in post_validation
    theme_dir = [utils.get_theme_dir(config['theme']), ]
  File "/usr/lib/python2.7/site-packages/mkdocs-0.16.3-py2.7.egg/mkdocs/utils/__init__.py", line 383, in get_theme_dir
    return os.path.dirname(os.path.abspath(theme.load().__file__))
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2291, in load
    return self.resolve()
  File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2297, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
ImportError: No module named material
```

### Expected behavior

I expected the mkdocs development server to start and to be able to see an mkdocs site using the Material theme.

### Actual behavior

I got the above error.

### Steps to reproduce the bug

1. Create an mkdocs.yml file:

```yaml
site_name: Foo
theme: material
```

2. Run the Docker image:

```bash
docker run --rm -it -p 8000:8000 -v `pwd`:/docs squidfunk/mkdocs-material
```

### Package versions

* Docker image:

```
» docker pull squidfunk/mkdocs-material
Using default tag: latest
latest: Pulling from squidfunk/mkdocs-material
Digest: sha256:1d6c134374f22dab39fa1b94f132a45e99f60bd69ebb56bb813d7e1193e62946
Status: Image is up to date for squidfunk/mkdocs-material:latest
```

### Project configuration

``` yaml
site_name: Foo
theme: material
```

### System information

* OS: OS X

**Comment by mattbostock** [comment]
The `squidfunk/mkdocs-material:1.5.1` Docker image works for me but not `squidfunk/mkdocs-material:1.5.2` or later.

**Comment by squidfunk** [comment]
I just pulled the latest image on OS X and it works perfectly. Very strange. Somebody else experiencing this problem?

**Comment by squidfunk** [comment]
You're right, managed to reproduce it. Will provide a fix asap.

**Comment by squidfunk** [comment]
Fixed in #279, also added a smoke test for the image. Thanks for reporting, that should definitely not happen!

**Comment by mattbostock** [comment]
Thanks for the quick fix! 👍
